CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.30.0)
   cmake_policy(SET CMP0167 OLD)
 endif()
 
+# use gnu standard install directories
+include(GNUInstallDirs)
+set(ARBORX_INSTALL_PACKAGEDIR "${CMAKE_INSTALL_LIBDIR}/cmake/ArborX" CACHE PATH "Install location of CMake target files")
```

Comment:
```suggestion
set(ARBORX_INSTALL_PACKAGEDIR "${CMAKE_INSTALL_LIBDIR}" CACHE PATH "Install location of CMake target files")
```

Reply:
I would also probably rename `ARBORX_INSTALL_PACKAGEDIR` to `ARBORX_INSTALL_CMAKEDIR`